src/playermat/PlaymatApi.ttslua

```diff
@@ -178,6 +178,15 @@ do
     end
   end
 
+  -- Returns the resource counter amount for the requested playermat
+  PlaymatApi.getResourceCount = function(matColor)
+    local resources = 0
+    for _, mat in ipairs(internal.getMatForColor(matColor)) do
+      resources = resources + mat.call("getResourceCount")
+    end
+    return resources
+  end
+
```

Comment:
Is there a specific reason for returning the sum of all resources instead of the specific values of the mats?